docs/indexer/build/manifest/chain-specific/cosmos.md

```
@@ -0,0 +1,317 @@
+          kind: cosmos/MessageHandler
+          filter:
+            type: "/cosmwasm.wasm.v1.MsgExecuteContract"
+            # Filter to only messages with the provide_liquidity function call
+            #contractCall: "provide_liquidity" # The name of the contract function that was called
+            #values: # A set of key/value pairs that are present in the message data
+            #contract: "juno1v99ehkuetkpf0yxdry8ce92yeqaeaa7lyxr2aagkesrw67wcsn8qxpxay0"
+```
+
+:::
+
+### Tested and Supported networks
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)